src/main/java/com/targomo/client/api/enums/MultiGraphLayerType.java

```diff
@@ -7,30 +7,31 @@
 
 import java.util.stream.Stream;
 
+import static com.targomo.client.api.enums.MultiGraphDomainType.*;
+
 /**
  * The currently implemented layer types that are supported by the multi graph request.
  */
 public enum MultiGraphLayerType {
 
-    NODE                (Constants.KEY_MULTIGRAPH_LAYER_TYPE_NODE,                  false,  false,  false),
-    EDGE                (Constants.KEY_MULTIGRAPH_LAYER_TYPE_EDGE,                  true,   false,  false),
-    TILE                (Constants.KEY_MULTIGRAPH_LAYER_TYPE_TILE,                  true,   true,   false),
-    TILE_NODE           (Constants.KEY_MULTIGRAPH_LAYER_TYPE_TILE_NODE,             false,  true,   false),
-    HEXAGON             (Constants.KEY_MULTIGRAPH_LAYER_TYPE_HEXAGON,               true,   true,   false),
-    HEXAGON_NODE        (Constants.KEY_MULTIGRAPH_LAYER_TYPE_HEXAGON_NODE,          false,  true,   false),
-    GEOMETRY_STATISTICS (Constants.KEY_MULTIGRAPH_LAYER_TYPE_GEOMETRY_STATISTICS,   false,  false,  true),
-    TILE_STATISTICS     (Constants.KEY_MULTIGRAPH_LAYER_TYPE_TILE_STATISTICS,       false,  true,   true);
+    IDENTITY            (Constants.KEY_MULTIGRAPH_LAYER_TYPE_IDENTITY,          false, MultiGraphDomainType.values()),
+    TILE                (Constants.KEY_MULTIGRAPH_LAYER_TYPE_TILE,              true,  EDGE, NODE),
+    HEXAGON             (Constants.KEY_MULTIGRAPH_LAYER_TYPE_HEXAGON,           true,  EDGE, NODE),
+    CUSTOM_GEOMETRIES   (Constants.KEY_MULTIGRAPH_LAYER_TYPE_CUSTOM_GEOMETRIES, false, MultiGraphDomainType.values());
+
+    private final String key;
+    private final boolean geometryMerge; //TODO probably needs to be renamed -
```

Comment:
TODOs still open? If yes, would probably makes sense to create issues for it instead.